apps/antalmanac/src/components/Calendar/Toolbar/CustomEventDialog/CustomEventDialog.tsx

```diff
@@ -226,8 +226,8 @@ function CustomEventDialogs(props: CustomEventDialogProps) {
                         {disabled
                             ? 'Schedule and day must be checked'
                             : props.customEvent
-                            ? 'Save Changes'
-                            : 'Add Event'}
+                              ? 'Save Changes'
+                              : 'Add Event'}
```

Comment:
Could you remove this diff?
You might need to commit with `git commit --no-verify` to prevent the linter from acting up. Or `git commit --empty` to allow empty changes.